content/blog/wingingit/winging-it-01.md

```diff
@@ -0,0 +1,45 @@
+---
+title: CSS Cascade Layers
+episode: 1
+author:
+  - miriam
+  - ed
+  - stacy
+date: 2023-07-20
+tags:
+  - CSS
+  - Cascade Layers
+length: 40 mins
+image:
+  src: winging-it/winging-it-1.jpg
+media:
+  - iframe: https://www.youtube.com/embed/FwICaSE8iuY
+summary: |
+  If you've ever found yourself in a specificity war, you'll understand how
+  important having control over style priority can be. During our conversation,
+  we discussed what CSS Layers are and how you can use them in your project.
+---
+
+{% import "embed.macros.njk" as embed %}
+
+{{ embed.figure(
+  data=media
+) }}
```

Comment:
@stacyk Add a caption here?

Reply:
I removed it intentionally since what I had was a duplicate of the post title and the screen reader will narrate the video title so I wasn't sure it caption is really needed here. 
![Screenshot 2023-07-20 at 2 29 57 PM](https://github.com/oddbird/oddleventy/assets/1581694/2fa3c9de-7610-47b4-a624-a36daac148f9)